opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -284,14 +322,19 @@ class Translator {
     /**
      * Internal helper method which makes the actual request to the wink service
      */
-    _makeRequest(path, method, content) {
+    _makeRequest(path, method, content, includeBearerHeader) {
+       
+        var addBearerHeader = (typeof includeBearerHeader !== 'undefined') ?  includeBearerHeader : true;
```

Comment:
yeah that's what the initial implementation was based on but turns out that the version of js we are using (am not sure how I can figure that out yet) isn't ECMA6.. we'll need to follow up offline on why we aren't using the latest and greatest yet.